scripts/get_geth.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# downloads and installs a pre-built Geth framework with an i386 slice
+
```

Comment:
add `set -e`
https://www.gnu.org/software/bash/manual/bashref.html#The-Set-Builtin

Reply:
this will case the script to exit immediately on any failures

Reply:
we should also add this to all sh/bash in this repo